### ISSUE TITLE
Out of topic

### ISSUE THREAD

**Issue by jobdiogenes** [rinaldi6109/CLT#1]
Hello,

Sorry to write here, but its the only way I found.
I am trying to use (https://github.com/fgborges/cheeriogs) in a Google Apps Script, I saw a message from you that works like a charm, but when I try to use (by key) lib does not load and show many errors. I make an Issue there, but seem that developer does not more work with it. 
How you use?
I also don't see in the repository the source code that Readme say that are an update. 

If you could help me in any way I'm very gratefully 

Jobdiogenes at gmail

**Comment by rinaldi6109** [comment]
What have you done exactly?
Write down here using a numberder list.
Moreover, copy and paste the errors log and make a screenshot of it.

**Comment by jobdiogenes** [comment]
Em sex., 22 de nov. de 2019 às 12:07, Antonio Rinaldi <
notifications@github.com> escreveu:

> What have you done exactly?

Write down here using a numberder list.
>

   1. Create a Google Apps Script
   2. Add to Library Resources the Key:
   1ReeQ6WO8kKNxoaA_O0XEQ589cIrRvEBA9qcWpNqdOP17i47u6N9M5Xh0
   3. Selected Last Version Available
   4. In Google Apps Code call:  var q = Cheerio.load(html)
   5. Run or Debug.



> Moreover, copy and paste the errors log and make a screenshot of it.
>
> From Stackdriverlog Run:
...
 at [unknown function](Code:298)
 at [unknown function](index:3745)
 ...


From Debug:
[image: image.png]


> On Fri, Nov 22, 2019 at 12:55 PM Job Diógenes Ribeiro Borges <

**Comment by rinaldi6109** [comment]
> 4. In Google Apps Code call: var q = Cheerio.load(html)

Have you written a line such as the following before?

const html = UrlFetchApp.fetch("https://en.wikipedia.org");

**Comment by jobdiogenes** [comment]
Yes I do. I also test with html = "<html> ....". But get errors while
library load.

Em sex., 22 de nov. de 2019 às 15:25, Antonio Rinaldi <
notifications@github.com> escreveu:

**Comment by rinaldi6109** [comment]
Share your workbook with me. I will continue the conversation inside gmail.
